modules/menuItems.js

```diff
@@ -547,6 +549,20 @@ let menuTempl = function (webviews) {
         });
     }
 
+    devToolsMenu.push({
+        type: 'separator'
+    },
+    {
+        label: i18n.t('mist.applicationMenu.develop.enableSwarm'),
+        enabled: true,
+        checked: [SwarmState.Enabling, SwarmState.Enabled].indexOf(global.store.getState().settings.swarmState) > -1,
+        type: 'checkbox',
+        click() {
+            store.dispatch(toggleSwarm());
+            store.dispatch(toggleSwarmOnStart());
```

Comment:
I like the idea of having `toggleSwarm` take care of the `onStart` stuff. From a dev's perspective I think It would be nice not to have to care/know about this implementation detail at this level. What do you think?